include/event_dbus_monitor.hpp

```diff
@@ -12,6 +13,7 @@ namespace crow
 namespace dbus_monitor
 {
 
+// NOLINTBEGIN(cppcoreguidelines-avoid-non-const-global-variables)
```

Comment:
Why do you need this?

Reply:
We can't move these to non-global variables?

Reply:
This is how our existing code designed, it needs more rework to avoid global variables.
this is followed in some places in upstream as well @gtmills 
https://github.com/openbmc/bmcweb/blob/master/include/image_upload.hpp#L21